tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -54,7 +54,7 @@ async function createZipFile(inputPath) {
 describe('firefox schema import', () => {
   // Skip the Firefox schema import tests on windows.
   if (process.platform === 'win32') {
-    // eslint-disable-next-line jest/no-disabled-tests
+    // eslint-disable-next-line jest/no-disabled-tests, no-undef
```

Comment:
@wagnerand sigh, that's annoying, do we know why isn't recognized anymore as a jest global?
(as it was when we changed that line for the last time in https://github.com/mozilla/addons-linter/commit/50c4a1e5861593c73293ce79e32077735dbaecc0)

I think that I would slightly prefer a more explicit:
```
// Explicitly retrieve Jest helper function from global.
const { pending } = global;
```

added right after the imports in this file (and remove the no-undef).

Reply:
caca6fe6

Reply:
Probably https://github.com/jest-community/eslint-plugin-jest/commit/a0e2bc526c5c22bcf4d60160242b55d03edb571d#diff-59236671f7bebb75fea8f66e85a9da68